src/errors.ts

```diff
@@ -9,24 +9,10 @@ export class SerializationError extends Error {
   }
 }
 
-export class DeserializationError extends Error {
-  constructor(message: string) {
-    super(message);
-    this.message = message;
-    this.name = this.constructor.name;
-    if (Error.captureStackTrace) {
-      Error.captureStackTrace(this, this.constructor);
-    }
-  }
-}
+export class SerializationError extends StacksTransactionError {}
```

Comment:
Custom JS error classes have wonky behavior. 

At a minimum, the extended error base class (`StacksTransactionError` in this case) must have `this.name = this.constructor.name;` otherwise the name string `SerializationError` will not appear in certain situations. 

Additionally, it's helpful for Node.js apps when custom errors include `Error.captureStackTrace?.(this, this.constructor);` in the constructor.

Reply:
There are other issues around minifying - if `this.name = this.constructor.name`, then often the exception thrown in production will be some single-letter variable.

Reply:
I reverted the changes for now.